docs/Contributing/dev-env.rst

```diff
@@ -106,7 +106,7 @@ On Linux, the easiest way to install KDevelop is with a command such as this:
 
 .. code-block:: sh
 
-    $ sudo apt install kdevelop clang-format-11
+    $ sudo apt install kdevelop clang-format-17 git-clang-format17 python3-sphinx
```

Comment:
What's are the new minimum required distro versions? `git-clang-format17` doesn't exist in Ubuntu 24.04 so I assume it's not in the 22.04 either.

We should update `compile.rst` accordingly. I'm not sure we want to bump requirements right before a release, so maybe this should come after 3.1?